test/unit/jest.config.js

```diff
@@ -8,6 +8,8 @@ const transpiledPackageNames = glob( 'packages/*/src/index.js' ).map(
 	( fileName ) => fileName.split( '/' )[ 1 ]
 );
 
+process.env.TZ = 'America/New_York';
```

Comment:
This sort of things should be part of the setup, rather than the config:
https://github.com/WordPress/gutenberg/blob/2293b3ba38993fdc5d49ffe08467ba2e0c6aec47/test/unit/jest.config.js#L20-L22

Reply:
Makes sense. I'll setup something more granular for test's timezone.